### PR TITLE
Adds test for widemul, div, rem in int.jl

### PR DESCRIPTION
**Pull request by mronian**
PR from a new branch #15465

### CODE REVIEW

File: test/int.jl [hunk 1]
```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+i=Int64(2)^63-1
```
Comment (tkelman) [inline test/int.jl]: this might be better as a `let i = Int64(2)^63, k = widemul(i,i)`
block since these are common variable names

**Comment by mronian** [comment]
The build is giving an error for 'make check whitespace'. Is that because of my code?

File: test/int.jl [hunk 2]
```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+let i=Int64(2)^63-1, k=widemul(i,i)
+
+    @test widemul(i,i)==85070591730234615847396907784232501249
+    
+    local j=div(k,2)
```
Comment (tkelman) [inline test/int.jl]: `local` isn't necessary here. the `check-whitespace` failure is due to the indent on the empty next line

File: test/int.jl [hunk 3]
```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+let i=Int64(2)^63-1, k=widemul(i,i)
+
```
Comment (tkelman) [inline test/int.jl]: the extra lines aren't really necessary here, it's readable enough without them

**Comment by mronian** [comment]
Thanks! 
I've made the changes and the tests passed now. Please review.

**Pull request merged**